.ci/setup.sh

```diff
@@ -41,6 +41,9 @@ pushd "${runtime_repo_dir}"
 sed -i 's/cc-agent.target/clear-containers.target/' create.go
 popd
 
+# Install libudev-dev required for go-udev vendor dependency
+sudo apt-get install libudev-dev
```

Comment:
Nevermind, this commit is completely fine, and we have to remove the one from the `tests` repo since it does not build the agent.